### DISCUSSION TITLE
Getting NPM error while creating react app

### DISCUSSION THREAD

**Discussion by Rishi-381** [community/community#31944] [npm]
Problem: On creating react app public, src and scripts folder is not created and also npm errors are poping while creating react app
NPM errors:
       npm ERR! code FETCH_ERROR
       npm ERR! errno FETCH_ERROR
       npm ERR! invalid json response body at https://registry.npmjs.org/@babel%2fhelper-skip-transparent-expression-wrappers reason: 
               Invalid response body while trying to fetch https://registry.npmjs.org/@babel%2fhelper-skip-transparent-expression-wrappers: 
               EBUSY: resource busy or locked, rename 'C:\Users\Asus\AppData\Local\npm-cache\_cacache\tmp\0e987701' -> 
               'C:\Users\Asus\AppData\Local\npm-cache\_cacache\content- v2\sha512\0a\4f\473b4caaf09b710f71b23c68d93ef797899aaa8562ddabac5f5a3004c4b20e5010d6c2720e96bde3ab51daf20fc39a1ff2d208e09caf8528a495aaae8cc3'
       npm ERR! A complete log of this run can be found in:
       npm ERR!     C:\Users\Asus\AppData\Local\npm-cache\_logs\2022-09-04T13_20_17_403Z-debug-0.log
Please someone help me on this issue.

what I tried:-
1. I uninstalled NodeJS and then reinstall it but didn't work
2. Following commands i have used to resolve the issue:
    1. npm cache clean --force
    2. npm rm -g create-react-app
        npm install -g create-react-app
        npx create-react-app my-app

**Comment by atomisadev** [comment]
Hi!

This isn't related to GitHub services, so please take it to the [`create-react-app` issues](https://github.com/facebook/create-react-app/issues) page.

Also, have you tried running `npm cache verify`?

**Reply by Rishi-381** [reply]
Thanks for the reply atomdevelops
no i haven't tried npm cache verify but what will happen if i do npm cache verify ?

**Reply by truongcongtrung2709** [reply]
Thanks, its work for me

**Reply by michael-duren** [reply]
npm cache verify worked nice

**Comment by Tungobhadra** [comment]
@Rishi-381 thanks, worked!
